app/controllers/api/v1/books_controller.rb

```diff
@@ -7,12 +7,13 @@ class BooksController < ApplicationController
       before_action :authenticate_user!
       def index
         @book = Book.all
-        render_paginated @book, each_serializer: BooksSerializer
+        render_paginated @book, serializer: BookSerializer
```

Comment:
https://github.com/Wolox/wor-paginate#basic-usage
```suggestion
        render_paginated Book
```